python/lsst/obs/decam/decamMapper.py

```diff
@@ -246,15 +246,8 @@ def std_raw(self, item, dataId):
         # Convert the raw DecoratedImage to an Exposure, set metadata and wcs.
         exp = exposureFromImage(item, logger=self.log)
         md = exp.getMetadata()
-        rawPath = self.map_raw(dataId).getLocationsWithRoot()[0]
-        headerPath = re.sub(r'[\[](\d+)[\]]$', "[0]", rawPath)
-        md0 = readMetadata(headerPath)
-        # extra keywords to copy to the exposure
-        for kw in ('DARKTIME', ):
-            if kw in md0.paramNames() and kw not in md.paramNames():
-                md.add(kw, md0.getScalar(kw))
         exposureId = self._computeCcdExposureId(dataId)
-        visitInfo = self.makeRawVisitInfo(md=md0, exposureId=exposureId)
+        visitInfo = self.makeRawVisitInfo(md=md, exposureId=exposureId)
```

Comment:
I am guessing that the reason the new translator tickled this is that the `exposureId` passed into the routine is now ignored and is there for backwards compatibility (all the dataId computations will be moved out of obs packages and handled by the metadata translator for gen3).

Reply:
Thanks, I hadn't realized it was ignored. I removed the parameter entirely now.